### PR TITLE
Spctr 2280 container block code refactor

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description
Removed get_fallback_number function call in frontend.css.php of container block,
have used the attributes directly without function call.

### Screenshots
https://d.pr/v/puEERB

### Types of changes
removal of get_fallback_number and passing the attrs directly without function call.

### How has this been tested?
1. have added null value to the attributes in editor.
2. using x-debug have checked the value of the attributes it take default values if we give null.
3. by giving a value to the attributes it take the current value we have passed.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: classes/class-uagb-block-module.php [hunk 1]
```diff
@@ -19,6 +19,7 @@
 	 */
 	class UAGB_Block_Module {
 
+
```
Comment (adi3890) [inline classes/class-uagb-block-module.php:22]: These changes should not appear as this is irrelevant.

File: includes/blocks/container/frontend.css.php [hunk 2]
```diff
@@ -7,13 +7,9 @@
...
  */
 
-$block_name                                  = 'container';
-$inner_content_custom_width_desktop_fallback = UAGB_Block_Helper::get_fallback_number( $attr['innerContentCustomWidthDesktop'], 'innerContentCustomWidthDesktop', $block_name );
-$inner_content_custom_width_tablet_fallback  = is_numeric( $attr['innerContentCustomWidthTablet'] ) ? $attr['innerContentCustomWidthTablet'] : $inner_content_custom_width_desktop_fallback;
-$inner_content_custom_width_mobile_fallback  = is_numeric( $attr['innerContentCustomWidthMobile'] ) ? $attr['innerContentCustomWidthMobile'] : $inner_content_custom_width_tablet_fallback;
-$width_desktop_fallback                      = UAGB_Block_Helper::get_fallback_number( $attr['widthDesktop'], 'widthDesktop', $block_name );
-$row_gap_desktop_fallback                    = UAGB_Block_Helper::get_fallback_number( $attr['rowGapDesktop'], 'rowGapDesktop', $block_name );
-$column_gap_desktop_fallback                 = UAGB_Block_Helper::get_fallback_number( $attr['columnGapDesktop'], 'columnGapDesktop', $block_name );
+$block_name                                 = 'container';
```
Comment (adi3890) [inline includes/blocks/container/frontend.css.php:10]: check if this $block_name is required further.

File: includes/blocks/container/frontend.css.php [hunk 3]
```diff
@@ -332,17 +328,17 @@
...
-				UAGB_Helper::get_css_value( $attr['boxShadowVOffset'], 'px' ) .
-				' ' .
-				UAGB_Helper::get_css_value( $attr['boxShadowBlur'], 'px' ) .
-				' ' .
-				UAGB_Helper::get_css_value( $attr['boxShadowSpread'], 'px' ) .
-				' ' .
-				$attr['boxShadowColor'] .
-				' ' .
-				$box_shadow_position_css,
+		UAGB_Helper::get_css_value( $attr['boxShadowHOffset'], 'px' ) .
```
Comment (adi3890) [inline includes/blocks/container/frontend.css.php:331]: this change is not required should be reverted

File: includes/blocks/container/frontend.css.php [hunk 4]
```diff
@@ -388,17 +384,16 @@
...
-																	' ' .
-																	UAGB_Helper::get_css_value( $attr['boxShadowBlurHover'], 'px' ) .
-																	' ' .
-																	UAGB_Helper::get_css_value( $attr['boxShadowSpreadHover'], 'px' ) .
-																	' ' .
-																	$attr['boxShadowColorHover'] .
-																	' ' .
-																	$box_shadow_position_css_hover;
-
+			' ' .
```
Comment (adi3890) [inline includes/blocks/container/frontend.css.php:387]: need to be reverted

File: includes/blocks/container/frontend.css.php [hunk 5]
```diff
@@ -411,22 +406,21 @@
...
```
Comment (adi3890) [inline includes/blocks/container/frontend.css.php:409]: needs to be reverted

File: classes/class-uagb-block-module.php (hunk 1 above)
Comment (adi3890) [reply classes/class-uagb-block-module.php:22]: All changes in this file should be reverted

Comment (rajmohanreddy774) [reply classes/class-uagb-block-module.php:22]: done with changes

Comment (rajmohanreddy774) [reply classes/class-uagb-block-module.php:22]: > These changes should not appear as this is irrelevant.

done with changes

File: includes/blocks/container/frontend.css.php (hunk 3 above)
Comment (rajmohanreddy774) [reply includes/blocks/container/frontend.css.php]: yes reverted

File: includes/blocks/container/frontend.css.php (hunk 4 above)
Comment (rajmohanreddy774) [reply includes/blocks/container/frontend.css.php]: done

File: includes/blocks/container/frontend.css.php (hunk 5 above)
Comment (rajmohanreddy774) [reply includes/blocks/container/frontend.css.php]: done

**Review by adi3890 — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**